CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+<!-- markdownlint-disable no-duplicate-heading -->
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- towncrier release notes start -->
+
+
+## v2.4.0
+
+- No longer support `Python 3.6` due to `asyncio.run` usage in the core.
```

Comment:
Wow I forgot to remove this line :(

Reply:
This is one of the reasons to have a proper release notes management :)